pagination.go

```diff
@@ -17,3 +17,10 @@ func (p ResultInfo) Next() ResultInfo {
 func (p ResultInfo) HasMorePages() bool {
 	return p.Page > 1 && p.Page < p.TotalPages
 }
+
+// DoneCount returns true for the last page and false otherwise.
+// Unlike Done(), this function uses the Total and PerPage fields to determine if
+// there are more pages to fetch.
+func (p ResultInfo) DoneCount() bool {
```

Comment:
we shouldn't need this; if this is needed, we aren't sending the correct v4 response envelope and we should have the team address it.

Reply:
I agree it shouldn't be needed. Unfortunately, there are other APIs besides deployments that don't return the full result_info information as well.

Reply:
can you shoot me a list please? this is breaking an API contract if it is on a listing endpoint.

Reply:
I've updated this to use the correct way. Once pages backend has support, then this will be all set.